src/index.ts

```diff
@@ -22,7 +22,11 @@ export interface Options extends axe.RunOptions {
 }
 
 export const injectAxe = () => {
-	cy.readFile<string>(require.resolve('axe-core/axe.min.js')).then((source) =>
+	const fileName =
+		require?.resolve && typeof require?.resolve === 'function'
```

Comment:
Wouldn't this also work?

```suggestion
		typeof require?.resolve === 'function'
```

Reply:
you'll get a typeError if you run typeof(undefined.resolve)

Reply:
It technically will work, sometimes JS does some weird stuff that i don't understand so i aired on the safe side and kept it. Ill change it if you'd like

Reply:
Actually, none of these are checking the existence of `require` and both would fail when `require` isn't defined. However, `require?.resolve &&` is unnecessary because this is exactly what `?.` does.